### PR TITLE
add nav for contact page (just redirects to team for now)

### PR DESCRIPTION
**Pull request by easherma**
Jannie mentioned today that it was hard to find contact info, so I added a link to the nav. We could eventually replace this with a dedicated contact/feedback page. 
I also updated the site copyright to 2018!

I had some weird dependency issues pulling this locally, and it affected the Gemfile. Feel free to disregard that..

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -142,7 +142,9 @@
               <li><a href='/blog/index.html'><i class='fa fa-pencil'></i> Blog</a></li>
               <li><a href='/projects.html'><i class='fa fa-rocket'></i> Projects</a></li>
               <li><a href='/about.html'><i class='fa fa-info-circle'></i> About</a></li>
+              <li><a href='/team.html' class='contact'><i class='fa fa-comment'></i> Contact</a></li>
```
Comment (derekeder) [inline _layouts/default.html]: we should have the name of the link be the same as the name of the page we're linking to. lets change it to team

also, we have not a lot of space on the upper nav. i think we'll need to collapse the links into a dropdown: https://getbootstrap.com/docs/3.3/components/#navbar

File: Gemfile.lock [hunk 2]
```diff
@@ -1,135 +1,252 @@
...
+    html-pipeline (2.7.1)
+      activesupport (>= 2)
       nokogiri (>= 1.4)
-    i18n (0.7.0)
-    jekyll (3.0.2)
-      colorator (~> 0.1)
+    http_parser.rb (0.6.0)
+    i18n (0.9.5)
+      concurrent-ruby (~> 1.0)
+    jekyll (3.7.3)
```
Comment (derekeder) [inline Gemfile.lock]: looks like we're upgrading to jekyll 3.7.3. we'll have to make sure this works on the live server when we deploy

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,4 @@
 .DS_Store
 _site
-vendor/
-.Gemfile.lock
-.bundle/config
+
+Gemfile.lock
```
Comment (derekeder) [inline .gitignore]: Gemfile.lock should stay in the repo: https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore#4151540

**Comment by derekeder** [comment]
@easherma do you still want to add this? if not, we should close

**Comment by easherma** [comment]
Oh I thought we had merged this in!

**Comment by derekeder** [comment]
This is 6mo stale now. Closing.

**Pull request closed without merging**